scripts/kube-init.sh

```diff
@@ -71,7 +71,7 @@ echo "Checking docker service"
 sudo docker ps
 
 echo "Download Kubernetes CLI"
-wget -q -O kubectl "http://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl"
```

Comment:
What about minkube? It's 5 lines bellow...

Reply:
That's in a different bucket that hasn't been migrated behind the reverse proxy yet.

Reply:
Thanks!